### PR TITLE
Add image publish workflow for centos-7-predictable-nic

### PR DESCRIPTION
**Pull request by a-crate**
Publish template is still missing so it can't successfully push to ARLE, forgot I need this workflow.

### CODE REVIEW

File: daisy_workflows/build-publish/enterprise_linux/centos_7_predictable_nic.wf.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    "sbom_util_gcs_root": {
+      "Value": "",
+      "Description": "The root gcs bucket for sbomutil, if using sbomutil to generate the SBOM."
+    }
+  },
+  "Steps": {
+    "build": {
+      "TimeOut": "60m",
+      "IncludeWorkflow": {
+        "Path": "${workflow_root}/image_build/enterprise_linux/centos_7.wf.json",
```
Comment (zmarano) [inline daisy_workflows/build-publish/enterprise_linux/centos_7_predictable_nic.wf.json:40]: This should be the new one

Comment (a-crate) [reply daisy_workflows/build-publish/enterprise_linux/centos_7_predictable_nic.wf.json]: Woops. Fixed.

**Comment by a-crate** [comment]
Added a publish template that goes only to bct-prod-images

**Pull request merged**